Taskfile.yml

```
@@ -98,21 +98,16 @@ tasks:
   test:
     desc: Runs test suite
     aliases: [t]
-    deps: [install]
+    sources:
+      - "**.*.go"
```

Looks like a typo, no?

```suggestion
      - "**/*.go"
```

---

Yep, thanks 😅